src/style.css

```diff
@@ -1 +1,97 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: local('Avenir');
```

Comment:
Ensure that the local font 'Avenir' is available on the user's system. If not, consider providing a fallback web font.